test/data/case5_dc.m

```diff
@@ -0,0 +1,65 @@
+% NESTA v0.6.0
```

Comment:
case5_dc includes losses in the line model (case_3 is lossless) and is included in the pf.jl and opf.jl test scripts.